apps/ui/components/reservation-unit/QuickReservation.tsx

```diff
@@ -459,6 +465,23 @@ const QuickReservation = ({
     return null;
   }
 
+  const lastPossibleReservationDate =
+    reservationUnit.maxReservationDuration != null &&
```

Comment:
Should it be `reservationUnit.reservationsMaxDaysBefore` instead of `reservationUnit.maxReservationDuration` ?